chef/cookbooks/crowbar/templates/default/crowbar-delete-pacemaker-resources.sh.erb

```diff
@@ -29,6 +29,13 @@ rm -f $UPGRADEDIR/crowbar-delete-pacemaker-resources-failed
 
 <% if @use_ha %>
 
+log "Resetting rabbitmq cluster..."
+rabbitmq_nodes=$(rabbitmqctl cluster_status | tr -d '\n' | sed -r -e 's/\s//g' -e 's/.*\{disc,\[//' -e 's/\].*//' -e "s/'//g" -e 's/,/ /g')
```

Comment:
We do it for those scripts ... see e.g. https://github.com/crowbar/crowbar-core/blob/master/chef/cookbooks/crowbar/recipes/prepare-upgrade-scripts.rb#L115